Procfile

```diff
@@ -1,4 +1,5 @@
 thin:                      bundle exec rails s -p $PORT
-sidekiq:                   bundle exec sidekiq -q miq_bot
+sidekiq:                   bundle exec sidekiq -q miq_bot, -q travis_event -L log/sidekiq.log
 issue_manager:             bundle exec ruby lib/bot/miq_bot.rb
+travis_listener:           bundle exec rails runner lib/travis_event/listener.rb
```

Comment:
launch the separate travis event listener process that waits for events from travis for the subscribed repos and sends any events to any event handlers to decide if they can process it.